apps/platform/src/app/(main)/(settings)/settings/@invites/page.tsx

```diff
@@ -44,21 +49,39 @@ function InvitesPage(): React.JSX.Element {
           <Skeleton className="h-14 max-w-3xl" />
         </div>
       ) : (
-        <div className="max-w-3xl flex flex-col sm:items-center justify-between gap-y-6">
-          {workspaceInvitations.map((invitation) => (
-            <div className="w-full" key={invitation.workspace.slug}>
-              <InviteCard
-                invitation={invitation}
-                key={invitation.workspace.slug}
-                setWorkspaceInvitations={setWorkspaceInvitations}
-                workspaceInvitations={workspaceInvitations}
-              />
+        <div className="flex max-w-3xl flex-col justify-between gap-y-6 sm:items-center">
+          {workspaceInvitations.length > 0 ? (
```

Comment:
take `workspaceInvitations.length > 0` and assign it in a const called `hasInvitations` this will keep the code clean